src/Symfony/Bundle/FrameworkBundle/Resources/config/translation_providers.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Translation\Bridge\Crowdin\Provider\CrowdinProviderFactory;
+use Symfony\Component\Translation\Bridge\Loco\Provider\LocoProviderFactory;
+use Symfony\Component\Translation\Bridge\Lokalise\Provider\LokaliseProviderFactory;
+use Symfony\Component\Translation\Bridge\PoEditor\Provider\PoEditorProviderFactory;
+use Symfony\Component\Translation\Provider\NullProviderFactory;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('translation.provider_factory.null', NullProviderFactory::class)
+            ->args([
+                service('http_client'),
```

Comment:
since all providers need http_client, FrameworkExtention should throw a LogicException when it's not there

Reply:
Isn't it the default here? I don't find a way to explicit it.

Reply:
a call to `$willBeAvailable` in FrameworkExtension?